### PR TITLE
Add NPR support

### LINKED ISSUE #107: NPR support
Now that NPR has an HTML-based audio player, it'd be great to get support for that.  I may implement this if I get a couple hours.

### PR DESCRIPTION
**Pull request by bdukes**
I still need to add links & icon, but this is functional.

Fixes #107

### CODE REVIEW

**Comment by bdukes** [comment]
Alright, I've added the NPR icon

File: package.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
   "readmeFilename": "README.md",
   "gitHead": "eb6f38decdf15fc18625bf60a602db2937feb229",
   "scripts": {
-    "build": "$(npm bin)/grunt",
-    "release": "$(npm bin)/grunt release",
-    "watch": "$(npm bin)/grunt watch"
```
Comment (cgravolet) [inline package.json]: Are you on Windows? Is that why this was changed?

Comment (bdukes) [inline package.json]: Yes, I am on Windows.  Sorry, I didn't realize I had checked that change in (though I _think_ that should be a safe change to make the project work better cross-platform).

Comment (cgravolet) [inline package.json]: I guess I overlooked the fact it wouldn't work cross-platform, but I didn't want globally installing grunt to be a dependency. I wonder if there is a cross-platform way to achieve that... To the internets!

Comment (bdukes) [inline package.json]: NPM scripts run with `./node_modules/.bin` added to the PATH, so it will use the locally installed versions automatically.

Comment (cgravolet) [inline package.json]: Learned something new, thanks! 👍

**Pull request merged**